Jenkinsfile

```diff
@@ -470,5 +471,45 @@ pipeline {
                 }
             }
         }
+        stage('aarch64') {
+          when {
+            beforeAgent true
+            expression { params.aarch64 }
+          }
+          agent { label 'aarch64 && packet' }
+          steps {
+            sh '''
+              GITCOMMIT=$(git rev-parse --short HEAD)
+
+              docker build --rm --force-rm --build-arg APT_MIRROR=cdn-fastly.deb.debian.org -t docker-aarch64:$GITCOMMIT -f Dockerfile .
+
+              docker run --rm -t --privileged \
+                -v "$WORKSPACE/bundles:/go/src/github.com/docker/docker/bundles" \
+                --name docker-pr-aarch64$BUILD_NUMBER \
+                -e DOCKER_GRAPHDRIVER=vfs \
+                -e DOCKER_EXECDRIVER=native \
```

Comment:
Yes. 
A lot of improvements had been made on master branch since I wrote the script in the new stage "aarch64".
Thanks for correcting them in the new PR.